src/scrapers/pelitorrentScraper.ts

```diff
@@ -0,0 +1,151 @@
+import { load } from 'cheerio';
+import { slugify } from '../lib/strings';
+import Scraper from './scraper';
+import { Magnet, ScraperRequest } from '../interfaces';
+
+const SOURCE = 'PeliTorrent';
+const DEFAULT_URL = 'https://torrenflix.com';
+
+export class PelitorrentScraper extends Scraper {
+
+  baseUrl: string = process.env.BASE_URL || DEFAULT_URL;
+
+  constructor() {
+    super('pelitorrent');
+  }
+
+  protected processMessage(message: ScraperRequest): Promise<Magnet[]> {
+    if (!message.title) throw new Error('Title is required');
+    if (message.seasonNum && message.episodeNum) {
+      return this.getEpisodeLinks(
+        message.spanishTitle || message.title,
+        message.year,
+        message.seasonNum,
+        message.episodeNum,
+      );
+    } else {
+      return this.getMovieLinks(
+        message.spanishTitle || message.title,
```

Comment:
I've found that this site uses English title by default. Did you verified it works with Spanish title? E.g: Interstellar

Reply:
I didn't notice that, I think it is a bit inconsistent given the tests I've made. 
Interstellar is both Interstellar in Spanish and English.
But here are some examples that are in Spanish / English for no apparent reason:

[Summer camp](https://www.imdb.com/title/tt26928684/) -> [Summer Camp in Torrenflix](https://www.torrenflix.com/movies/summer-camp-2/)
[The abandon](https://www.imdb.com/title/tt10273738) -> [Perdido in Torrenflix](https://www.torrenflix.com/movies/perdido-2/)
[Tell me lies](https://www.imdb.com/title/tt13018148) -> [Tell me lies in Torrenflix] (https://www.torrenflix.com/series/dime-mas-mentiras/)
[Alien Resurrection](https://www.imdb.com/title/tt0118583) -> [Alien: Resurrección in Torrenflix](https://www.torrenflix.com/movies/alien-resurreccion/)

So with this in mind, the solution was to scrape the English title first and if nothing found proceed with the Spanish title (see the `processMessage` method in the new `torrenflixScraper.ts` file)